contracts/wrapper/WAuraBooster.sol

```diff
@@ -163,8 +167,6 @@ contract WAuraBooster is IWAuraBooster, BaseWrapper, ReentrancyGuardUpgradeable,
         uint256 balRewardPerToken = IRewarder(auraRewarder).rewardPerToken();
         id = encodeId(pid, balRewardPerToken);
 
-        _validateTokenId(id);
```

Comment:
My observation here is that the validation prevents the sender from minting more units of a previously minted token, are we okay with allowing that happen now?

Reply:
Your observation is correct but this is the exact reason we need to remove this validation. The whole reasoning for tokenId's is to store the starting point of `rewardPerToken` for each position. If two users happen to enter the position immediately after one another there is a chance that `rewardPerToken` will be the same. All accounting is done within the bank stating how many of the wrapper tokens a user has, which ultimately determines all reward calculations within the contract.

Reply:
Ah, I get it now, thanks for clarifying.